qiskit_optimization/algorithms/cobyla_optimizer.py

```diff
@@ -110,9 +110,15 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
         """
         self._verify_compatibility(problem)
 
+        # we deal with minimization in the optimizer, so turn the problem to minimization
+        from ..converters.maximize_to_minimize import MaximizeToMinimize
```

Comment:
Similar comment about inline import - so I will not comment on any more that might exist, suffice to say the above comment on inline import versus being at head of file applies.

Reply:
Fixed imports